_config.yml

```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
 
 # Build settings
 markdown: kramdown
+
+team:
+  - name: Ben
+    image: ben.png
+    url: https://twitter.com/socksy
+  - name: Franka
+    image: franka.jpg
+    url: http://franka.tech
+  - name: Danielle
+    image: dani.jpg
+    url:
+  - name: Dave
+    image: dave.png
+    url: http://daveliepmann.com
+  - name: Johanna
+    image: johanna.jpg
+    url:
+  - name: Julia
+    image: julia.jpg
+    url:
+  - name: Lisa
+    image: lislis.png
+    url: http://lislis.de
+  - name: Nils
+    image: nils.png
+    url:
+  - name: Saskia
+    image: saskia.jpg
+    url:
```

Comment:
http://saskialindner.com for me, thank you!

Reply:
omg I love it!! gonna have to got that flowing hand tree image for you soon (: